plugins/localization/src/components/language-switcher.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { ApplicationContext } from '../interfaces/application-context'
+import { useEffect } from 'react'
+import Gear from '../icons/gear'
+const context: ApplicationContext = require('@builder.io/app-context').default
+const pluginId = 'language-switcher'
+
+declare global {
+  interface Window {
+    languageSettingsTrigger: () => Promise<void>
+  }
+}
+
+const LangugeSwitcher = () => {
+  const [locales, setLocales] = useState<
+    {
+      code: string
+      name: string
+    }[]
+  >()
+  const onLoad = async () => {
+    const currentUser = await context.user.getUser(context.user.id)
+    //@ts-ignore
+    const currentOrg = await context.user.organization
+    //@ts-ignore
+    const pluginSettings = await (await currentOrg.value.settings.plugins).get(
+      pluginId
+    )
+    console.log({ pluginSettings })
```

Comment:
Logs, in here and several other files

Reply:
removed logging except for caught errors